codility_training/lessons.lesson02.Arrays.CyclicRotation/khw970421.js

```diff
@@ -8,3 +9,19 @@ function solution(A, K) {
     }
     return b;
 }
+//방법2
+function solution1(A, K) {
+    const size = A.length;                          //배열의 크기 size
+    const shift = (A.length>K)?(K):K%A.length;      //움직인 횟수 shift K의값이 크기보다 클경우 반복되는 횟수 나머지만 계산
```

Comment:
```suggestion
    const shift = K % size;      //움직인 횟수 shift K의값이 크기보다 클경우 반복되는 횟수 나머지만 계산
```
- 앞서 선언한 size를 사용하기
- 어차피 K % szie = K